usaspending_api/api_contracts/contracts/v2/download/assistance.md

```diff
@@ -0,0 +1,34 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Assistance Download [/api/v2/download/assistance/]
+
+## POST
+
+Returns a zipped file containing contract award data
+
++ Request (application/json)
+    + Attributes
+        + `award_id`: `ASST_NON_12FA00PY52375933_12D2` (required, string)
++ Response 200 (application/json)
+    + Attributes
+        + `results` (AssistanceDownloadResponse)
+
+# Data Structures
+
+## AssistanceDownloadResponse (object)
++ `total_size`: 35.055 (number, nullable)
```

Comment:
Are `total_size`, `total_rows`, `total_columns`, `message` not always provided by the response?

Reply:
message is usually null, but the totals should be populated